### PR TITLE
[FIX] hr_expense: Expense onboarding comments followup

### PR DESCRIPTION
**Pull request by hsal-odoo**
This commit resolves the unresolved comments mentioned on the expense onboarding PR https://github.com/odoo/odoo/pull/166321

task-3919542

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by JulienVR — COMMENTED** [review]
We will merge this in 17.3, could you retarget please ?
Note that changing a view in stable is not allowed normally but it has to be done anyway (since the view is broken for now). It should not be a problem since nobody is using the 17.3 for now. :+1:

File: addons/sale_expense/views/product_view.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
                     </div>
                 </group>
             </xpath>
-            <xpath expr="//field[@name='standard_price']" position="before">
+            <xpath expr="//label[@for='standard_price']" position="before">
```
[the remark this answers is not in the document]
Comment (hsal-odoo) [reply addons/sale_expense/views/product_view.xml:18]: It was Laura's catch actually, I just fixed it :D

[the remark this answers is not in the document]
Comment (JulienVR) [reply addons/sale_expense/views/product_view.xml:18]: nice catch @Laura-labe then ahah

**Review by JulienVR — APPROVED** [review]
Thanks a lot !
@robodoo r+

**Pull request closed without merging**